audit/audit.go

```diff
@@ -0,0 +1,111 @@
+package audit
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os/user"
+
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+	"go.mozilla.org/sops/logging"
+	"gopkg.in/yaml.v2"
+)
+
+var log *logrus.Logger
+
+func init() {
+	log = logging.NewLogger("AUDIT")
+	confBytes, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		log.WithField("error", err).Debugf("Error reading config")
+		return
+	}
+	var conf config
+	err = yaml.Unmarshal(confBytes, &conf)
+	if err != nil {
+		log.WithField("error", err).Debugf("Error unmarshalling config")
+		return
+	}
+	// Only create auditors if not running tests
+	// This is pretty hacky, but doing it The Right Way would require
+	// restructuring SOPS to use dependency injection instead of just using
+	// globals everywhere.
+	if flag.Lookup("test.v") == nil {
+		for _, pgConf := range conf.Backends.Postgres {
+			auditors = append(auditors, NewPostgresAuditor(pgConf.ConnStr))
+		}
+	}
+}
+
+const configFile = "/etc/sops/audit.yaml"
```

Comment:
This could probably be platform specific for folks running in windows? unsure if that's worth the effort...